src/flux/task.coffee

```diff
@@ -221,6 +221,13 @@ TaskConfig =
 
       result
 
+    getExternalStep: (id) ->
+      step = _.findWhere(@_steps[id], type: 'external_url')
+      unless step?.external_url.match(/^https?:\/\//)
```

Comment:
Is this not already accounted for when building an external link? I thought @pandafulmanda added the JS bit that prepends a link with `http://`